pkg/resources/python/resource_python_test.go

```diff
@@ -68,7 +66,7 @@ func TestAccPython_basic(t *testing.T) {
 		},
 		Steps: []resource.TestStep{{
 			ResourceName: rName,
-			Config:       fmt.Sprintf(providerBlock, org) + fmt.Sprintf(pythonBlock, rName, rName),
+			Config:       helper.NewProvider("clevercloud").SetOrganisation(org).String() + fmt.Sprintf(pythonBlock, rName, rName),
```

Comment:
You can also use your Resource helper to replace 
`fmt.Sprintf(pythonBlock, rName, rName)`